src/copyrightinserter.ts

```diff
@@ -191,15 +191,35 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.`]
         return line;
     }
 
+    private formatString(header: string, first_line: string, prefix: string, post: string): string {
+        var cur_prefix = first_line
+        var result : string = ""
+        for (const line of header.split("\n")) {
+            const new_line = cur_prefix + line
+            result += new_line.trimRight() + "\n"
```

Comment:
Appending to all lines of the header including last line creates unclear behavior for a caller of the method when `post` argument is used with \newline symbol too. Either the `first_line` argument should be made `prefix` and placed in a dedicated line and then `post` is placed in the dedicated line as well or, the first line of the header should be prefixed with `first_line` and the last line of the header should be suffixed with `post` (renamed to `last_line`).

Reply:
done